build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -364,7 +364,8 @@ public static BuildTargetResult PublishSharedFrameworkAndSharedHost(BuildTargetC
             var dotnetCli = DotNetCli.Stage0;
             var hostVersion = c.BuildContext.Get<HostVersion>("HostVersion");
             var sharedFrameworkNugetVersion = c.BuildContext.Get<string>("SharedFrameworkNugetVersion");
-            var hostFxrVersion = hostVersion.LockedHostFxrVersion.ToString();
+            //We provide a different locked hostFxr version for Windows (MSIs) instead of LockedHostFxrVersion for issue #1574. This we can update when we have the next binary change for the hostfxr.dll (from 1.0.1)
```

Comment:
I would suggest adding comment to the tune of "Get version to be used for MSI generation" to explain the overall purpose of this change.